draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+created. This balance will be initialized to zero at the NU6 activation block.
+
+<u>Coinbase Transaction Adjustments</u>
+
+For coinbase transactions, a specified proportion of the block subsidy will be
+added to the deferred development fund chain value pool balance and subtracted
+from the transparent transaction value pool. This subtraction will occur after
+block reward value is logically added to the transparent value pool. The exact
+proportions (numerator/denominator) of the block subsidy allocated to the
+deferred development fund must be defined to ensure the correct allocation at
+every block.
+
+<u>Proportion Specification</u>
+
+The proportions of the block subsidy allocated to the deferred development fund
+and any existing funding streams must add up to the intended total at every
+block. Before NU6 activation, the actual percentage allocated to the deferred
+development fund will be zero. After NU6 activation, |percentage| of the block
+reward will be allocated to the deferred development fund.
+
+<u>Backwards Compatibility</u>
+
+Full validators can implement the reserve consensus rules without NU gating.
+Prior to NU6 activation, the percentage allocated to the reserve pool can be
+set to zero or be defined as an empty set, similar to existing funding streams.
+Additionally, another implementation option would be to define a new type of
+funding stream that directs the value to the in-protocol development fund
+instead of constructing a UTXO. This approach allows for flexibility in
+handling the allocation and ensures compatibility with existing mechanisms.
+
+<u>Reserve Pool Mechanism</u>
+
+The reserve pool will be a container for the issued supply, distinct from
+existing pools. It will not be subdivided into individually spendable notes or
+coins. The lockbox mechanism ensures that funds are accumulated and securely
+stored until a future ZIP defines the disbursement mechanism.
+
+<u>Security and Integrity</u>
+
+Robust security measures must be implemented to protect the deferred
+development fund from unauthorized access or misuse. The integrity of the
+lockbox and its interaction with the Zcash protocol must be ensured.
+
+### Zcash Community Grants (ZCG)
+
+This slice of the Dev Fund is intended to fund independent teams entering the
+Zcash ecosystem, to perform major ongoing development (or other work) for the
+public good of the Zcash ecosystem, to the extent that such teams are available
+and effective. The Zcash Community Grants (ZCG) Committee is given the
+discretion to allocate funds not only to major grants, but also to a diverse
+range of projects that advance the usability, security, privacy, and adoption
+of Zcash, including community programs, dedicated resources, and other projects
+of varying sizes.
+
+The funds SHALL be received and administered by the Zcash Foundation (ZF) or
+Financial Privacy Foundation (FPF). ZF or FPF MUST disburse them for grants and
```

Comment:
The choice of ZF or FPF here would need to be considered a parameter for the purposes of today's deadline; it needs to be decided upon by the next deadline, as it cannot be either-or in the final ZIP.

Reply:
Okay, understood. I believe it will be FPF, similar to what's reflected in Dodger's proposal. I will confirm and follow up in the next day or two.